packages/block-editor/src/components/block-vertical-alignment-control/test/index.js

```diff
@@ -3,6 +3,7 @@
  */
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { act } from 'react-test-renderer';
```

Comment:
@jsnajdr perhaps we could use the re-exported `act` from `@testing-library/react`?

Reply:
Oh yes, this was VSCode eagerly adding a random import statement as soon as I typed "act" 🙂